### PR TITLE
Eugene - bookshelf

### PR DESCRIPTION
**Pull request by yevhensydorov**

### CODE REVIEW

File: bookshelf.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    .then(response => response.json())
+    .then(processBooks) // call the processBooks function
+    .then(removeBtn) // call the removeBtn function
+}
+
+function processBooks(booksJSON) {
+	var ulTag = document.createElement("ul"); // create <ul> Tag
+
+	// This forEach loop creates several li tags
+	booksJSON.forEach(function(el){
```
Comment (40thieves) [inline bookshelf.js]: Nitpick: I think `book` would be a better name for this variable

**Comment by 40thieves** [comment]
I forgot to mention if you click through [here](https://travis-ci.org/CodeYourFuture/bookshelf-project/builds/310457024?utm_source=github_status&utm_medium=notification) you can see the output of the tests. Or you can run them yourself like we did with the previous tests

File: bookshelf.js (hunk 1 above)
Comment (yevhensydorov) [reply bookshelf.js]: done!

File: bookshelf.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			}
+		});
+	}
+
+	function removeBtn() {
+		var button = document.getElementById("fetch-books-btn");
+		button.removeEventListener("click", fetchBooks);
+		button.remove();
+	}
+}())
```
Comment (40thieves) [inline bookshelf.js]: You've created the IIFE, but not "exported" the functions by returning them. This means that they can't be accessed from the outer scope

**Pull request closed without merging**